packages/ag-charts-community/benchmarks/largeDataset.test.ts

```diff
@@ -7,7 +7,7 @@ import { addSeriesNodePoints, benchmark, setupBenchmark } from './benchmark';
 describe('large-dataset benchmark', () => {
     const ctx = setupBenchmark<AgCartesianChartOptions>('large-dataset');
 
-    benchmark('initial load', ctx, { expectedRelativeMB: 660, expectedCanvasCount: 9 }, async () => {
+    benchmark('initial load', ctx, { expectedRelativeMB: 760, expectedCanvasCount: 9 }, async () => {
```

Comment:
This is expected as we now generate more commands. We could optimise this by sharing the path between markers and translating